### PR TITLE
Fix typos related to conda stuff

### PR DESCRIPTION
**Pull request by NoraLoose**
This fixes three minor typos related to conda instructions on the website, including the link to the conda environment file.

### CODE REVIEW

File: _InstructionSiteUpdates.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ It is worth noting that we are using the `source` branch as the base for this re
 #### Environment
 
 There is a pre-configured conda environment in `environment.yml`.
-To use it, `conda create env -f=./environment.yml`, then activate with `conda activate ohw-site`.
+To use it, `conda env create -f=./environment.yml`, then activate with `conda activate ohw-site`.
```
Comment (ocefpaf) [inline _InstructionSiteUpdates.md:60]: Let's take this opportunity to change this to the full keyword argument b/c it is more readable.
```suggestion
To use it, `conda env create --file environment.yml`, then activate with `conda activate ohw-site`.
```

File: resources/prep/conda.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ The [package](https://github.com/Anaconda-Platform/nb_conda_kernels) that detect
 
 ## Conda on your own computer
 
-Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://github.com/oceanhackweek/ohw20-tutorials/blob/master/environment.yml) that we use, then `conda create -n <ENV NAME> --file environment.yml`
+Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://github.com/oceanhackweek/jupyter-image/blob/main/py-base/environment.yml) that we use, then `conda env create -n <ENV NAME> --file environment.yml`
```
Comment (ocefpaf) [inline resources/prep/conda.md:93]: This env file is Linux only. Let's keep the reference to the ohw20-tutorials and I'll add a proper env file there. We still need to change `master` to `main`.

Comment (NoraLoose) [reply resources/prep/conda.md:93]: Sounds good. The link was broken, but there may be a better link to use here, as you say. Thanks, Filipe!

Comment (emiliom) [reply resources/prep/conda.md:93]: We may not have a great option here, for now. The OHW20 env file includes both Python and R, which we may not want keep here. But to at least fix the broken link, here's the correct, updated link:
https://github.com/oceanhackweek/ohw-tutorials/blob/OHW20/environment.yml

**Comment by emiliom** [comment]
Thanks for submitting these fixes @NoraLoose !

File: resources/prep/conda.md (hunk 2 above)
Comment (NoraLoose) [reply resources/prep/conda.md]: Thanks @emiliom, I updated the link!

Comment (ocefpaf) [reply resources/prep/conda.md]: Just added a "canonical one" here: https://github.com/oceanhackweek/ohw-tutorials/pull/100

It should be in main so we can port from year to year. Also, I'm pretty sure that one from OHW20 is also platform specific. The one I added above should be platform agnostic, even on the new mac M1/2.

File: resources/prep/conda.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ The [package](https://github.com/Anaconda-Platform/nb_conda_kernels) that detect
 
 ## Conda on your own computer
 
-Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://github.com/oceanhackweek/ohw20-tutorials/blob/master/environment.yml) that we use, then `conda create -n <ENV NAME> --file environment.yml`
+Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://github.com/oceanhackweek/ohw-tutorials/blob/OHW20/environment.yml) that we use, then `conda env create -n <ENV NAME> --file environment.yml`
```
Comment (ocefpaf) [inline resources/prep/conda.md:93]: This is a permalink to the "raw" file. Safer download and the link should not change.

```suggestion
Conda may be used on your computer as well as the Hub. If you wish to install the same environment as the hub is running, after you install Conda, you can download the [`environment.yml`](https://raw.githubusercontent.com/oceanhackweek/ohw-tutorials/d51c880111305d7e345d98b7d8ccc031cbf0087e/environment.yml) that we use, then `conda env create -n <ENV NAME> --file environment.yml`
```

**Pull request merged**